content/applications/inventory_and_mrp/inventory/routes/strategies/removal.rst

```diff
@@ -1,222 +1,362 @@
-==================================================
-What is a Removal Strategy (FIFO, LIFO, and FEFO)?
-==================================================
-
-Usually, *Removal Strategies* are defined in picking operations to select the best products to
-optimize the distance for the worker, for quality control purposes, or to first move the products
-with the closest expiration date.
-
-When a product movement needs to be done, Odoo finds available products that can be assigned to
-the transfer. The way Odoo assigns these products depends on the *Removal Strategy* defined in
-the *Product Category* or on the *Location*.
+=====================================
+Removal Strategies (FIFO, LIFO, FEFO)
+=====================================
+
+For companies with warehouses, **removal strategies** determine which products are taken from the
+warehouse, and when. Removal strategies are typically defined for specific picking operations. This
+helps companies to select the best products, optimize the distance workers need to travel when
+picking items for orders, and account for quality control, such as moving products with expiration
+dates.
+
+When a product needs to be moved, Odoo finds available products that can be assigned to the
+transfer. The way Odoo assigns these products depends on the :guilabel:`Removal Strategy` defined in
+either the :guilabel:`Product Category` or the :guilabel:`Location`.
+
+To change the :guilabel:`Removal Strategy`, go to :menuselection:`Inventory app --> Configuration
+--> Locations` or :guilabel:`Product Categories`. Click on a :guilabel:`Location` or
```

Comment:
I decided on (and hope you agree) that the 'locations' and 'product categories' in the previous sentence should both be menu selections since they are part of the navigation drop downs (truly a menu selection). Clicking the location or category in this sentence, they are items you click in the dashboard view, as oppose to a navigation direction- so since they are different ways of selecting, I think everything in the previous sentence is a menu selection, and these are GUI's. If you disagree, no problem- I can change again- but that is the logic I am following (so my first gui for 'product categories' was incorrect).